src/server/sheets/sheets.ts

```diff
@@ -167,7 +179,7 @@ export async function extractSheetData(
     proficiencyBonus: processProficiency(data[1]),
     abilityBonuses: processAbilityBonuses(data[2]),
     saveBonuses: processSaveBonuses(data[3]),
-    checkBonuses: processCheckBonuses(data[4]),
+    checkBonuses: processCheckBonuses(data[4], data[2]),
     attackBonuses: processAttackBonuses(data[5]),
```

Comment:
It might be a little cleaner to merge the two maps rather than modifying the processCheckBonuses to re-process the ability check data,

i.e

// renamed processCheckBonuses -> processSkillBonuses  for clarity here
const skillBonuses = processSkillBonuses(data[4]);
const abilityBonuses = processAbilityBonuses(data[2]);

return {
  ...
  abilityBonuses: abilityBonuses,
  checkBonuses: new Map([...skillBonuses, ...abilityBonuses),
}

WDYT?

Reply:
Definitely better. Updated